test/mongodb.test.js

```diff
@@ -116,8 +117,30 @@ describe('mongodb connector', function() {
         done();
       });
     });
-  });
 
+    it('should skip connect phase (lazyConnect = true)', function(done) {
+      var ds = getDataSource({
+        host: '127.0.0.1',
+        port: 4,
+        lazyConnect: true,
+      });
+      initialize(ds, done);
```

Comment:
I'm not sure if calling `initialize` is good idea as the constructor of DataSource already triggers it.